linear_x_vel_turbo: 1.1  # Trubo velocity on Linear X
linear_x_vel_nomal: 0.5  # nomal velocity on Linear X
linear_y_vel_turbo: 1.1  # Trubo velocity on Linear Y
linear_y_vel_nomal: 0.5  # nomal velocity on Linear Y
linear_z_vel_turbo: 3.14 # Trubo velocity on Angular Z (PI)
linear_z_vel_nomal: 1.04 # nomal velocity on Angular Z (PI/3)
holonomic: true          # enable holonomic robot or not
